packages/spectral/src/index.ts

```diff
@@ -66,24 +70,30 @@ export const flow = <
 ): T => definition;
 
 /**
- * For information on writing Code Native Integrations, see
- * https://prismatic.io/docs/code-native-integrations/#adding-config-pages.
+ * This function creates a config wizard page object for use in code-native
+ * integrations. For information on code-native config wizards, see
+ * https://prismatic.io/docs/integrations/code-native/config-wizard/.
+ *
  * @param definition A Config Page type object.
  * @returns This function returns a config page object that has the shape the Prismatic API expects.
  */
 export const configPage = <T extends ConfigPage = ConfigPage>(definition: T): T => definition;
 
 /**
- * For information on writing Code Native Integrations, see
- * https://prismatic.io/docs/code-native-integrations/#adding-config-vars.
+ * This function creates a config variable object for code-native integrations.
+ * For information on writing code-native integrations, see
+ * https://prismatic.io/docs/integrations/code-native/config-wizard/#other-config-variable-types-in-code-native-integrations.
+ *
  * @param definition A Config Var type object.
  * @returns This function returns a standard config var object that has the shape the Prismatic API expects.
  */
 export const configVar = <T extends StandardConfigVar>(definition: T): T => definition;
 
 /**
- * For information on writing Code Native Integrations, see
- * https://prismatic.io/docs/code-native-integrations/#adding-config-vars.
+ * This function creates a data source-backed config variable for code-native
+ * integrations. For information on code-native data sources, see
+ * https://prismatic.io/docs/integrations/code-native/config-wizard/#data-sources-is-code-native-integrations.
```

Comment:
Any worry about these urls having to be maintained if those docs links ever change over time?

Reply:
We did a big sweeping change of docs structure in 2024 Q4. The game-plan going forward is to try our darndest not to change docs URLs going forward and, when necessary, add an HTTP redirect.